python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -6044,6 +6044,19 @@ def test_mode(self):
         with self.assertRaises(ValueError):
             psdf.mode(axis=2)
 
+        def f(index, iterator):
+            return ["3", "3", "3", "3", "4"] if index == 3 else ["0", "1", "2", "3", "4"]
+
+        rdd = self.spark.sparkContext.parallelize(
+            [
+                1,
+            ],
+            4,
```

Comment:
nit: can we just is one or two line some thing like:

```python
rdd = self.spark.sparkContext.parallelize([1], 4)
    .mapPartitionsWithIndex(f)
```

?? I suspect it's maybe adjusted by black script tho, 😂

Reply:
it was just reformated by the script😅